data/strings/strings.txt

```diff
@@ -11356,47 +11356,48 @@
+    bg = Опитайте да промените критериите за търсене или изтеглете региона на картата, в който търсите.
+    ca = Proveu de canviar els criteris de cerca o descarregueu la regió del mapa on esteu cercant.
+    cs = Zkuste zadat jiný hledaný výraz nebo si stáhněte mapu regionu, ve kterém hledáte.
+    da = Prøv en anden forespørgsel, eller download det kortområde, hvor du søger.
+    de = Versuchen Sie bitte eine andere Suchanfrage oder laden die Kartenregion herunter, in der Sie suchen.
+    el = Δοκιμάστε να αλλάξετε τα κριτήρια αναζήτησης ή να κατεβάσετε τον χάρτη της περιοχής στην οποία ψάχνετε.
+    es = Por favor, pruebe con otra consulta o descarga el mapa de la región donde buscas.
+    et = Proovi muuta otsingutingimusi või laadige alla kaardipiirkond, kus te otsite.
+    eu = Saiatu bilaketa-irizpideak aldatzen edo deskargatu bilatzen ari zaren mapa-eskualdea.
+    fa = معیارهای جستجوی خود را تغییر دهید یا منطقه نقشه را که در آن جستجو می‌کنید دانلود کنید.
+    fi = Yritä jotain toista hakusanaa tai lataa kartta-alue, jolta etsit.
+    fr = Veuillez essayer un autre mot-clé ou de télécharger la carte de la région où tu cherches.
+    he = נסה לשנות את קריטריוני החיפוש שלך או להוריד את אזור המפה שאתה מחפש.
+    hi = अपना खोज मानदंड बदलने का प्रयास करें या वह मानचित्र क्षेत्र डाउनलोड करें जहां आप खोज रहे हैं।
+    hu = Próbáljon másik kifejezést, vagy töltse le azt a térképes régiót, ahol keres.
+    id = Silakan coba kueri lainnya atau unduh wilayah peta tempat Anda mencari.
+    it = Prova con un'altra ricerca o a scaricare la mappa della regione in cui stai effettuando la ricerca.
+    ja = 検索条件を変更するか、検索している地図地域をダウンロードする。
+    ko = 검색 기준을 변경하거나 검색하려는 지도 지역을 다운로드해 보세요.
+    lt = Pabandykite pakeisti jūsų paieškos kriterijus arba atsisiųskite žemėlapio regioną, kuriame ieškote.
+    mr = तुमचा शोध निकष बदलण्याचा प्रयत्न करा किंवा तुम्ही जिथे शोधत आहात तो नकाशा प्रदेश डाउनलोड करा.
+    nb = Vennligst prøv et annet søk eller last ned kartregionen der du søker.
+    nl = Probeer een andere zoekopdracht of download de kaartregio waar je zoekt.
+    pl = Wpisz inne zapytanie lub pobierz mapę regionu, w którym szukasz.
+    pt = Por favor, tente outro termo de pesquisa ou descarrega a região do mapa onde estás a pesquisar.
+    pt-BR = Por favor, tente outro termo de busca ou faça o download da região do mapa onde você está pesquisando.
+    ro = Încearcă să modifici condițiile căutării sau descărcați harta regiunii în care căutați.
+    ru = Попробуйте ввести другое слово, или загрузить карту региона, в котором ищете.
+    sk = Vyskúšajte, prosím, iný hľadaný výraz alebo si stiahnite mapu regiónu, v ktorom hľadáte.
+    sv = Försök med en annan sökförfrågan eller ladda ner kartregionen där du söker.
+    sw = Jaribu kubadilisha vigezo vyako vya utafutaji au pakua eneo la ramani unapotafuta.
+    th = ลองเปลี่ยนเกณฑ์การค้นหาของคุณหรือดาวน์โหลดขอบเขตแผนที่ที่คุณกำลังค้นหา
```

Comment:
@organicmaps/translations please review.

Reply:
For our reference:
en = Try changing your search criteria or download the map region where you are searching.

Reply:
In languages that have different levels of formality (Japanese, French, Russian I think, Romanian etc.), what is the recommended way to address the user? I looked at other translations in Romanian and it's mixed, some are formal and some are informal. This particular translation is half informal, half formal. Also looking at the translations above I see French uses "tu" (informal) but German has "Sie" (formal).

Personally it feels that informal would work better in Romanian, but let me know if there are existing project guidelines about this.

Reply:
We stick to formal translations, looks like the auto-generated translations prefer informal for some reason.

Informal can be ok for a language, but then it should be consistently informal everywhere.

Looks like the source language for translations should not be English but the one with informal/formal differences.

Reply:
Here was it, now translations will be more formal: https://github.com/organicmaps/organicmaps/pull/8890